sys/dev/psci/smccc_arm64.S

```diff
@@ -64,6 +64,27 @@ __FBSDID("$FreeBSD$");
 .endmacro
 #endif
 
+/*
+ * XXX: TF-A on Morello systems depends on an almighty cap in DDC_EL0.
+ */
+#if __has_feature(capabilities)
+.macro	reset_ddc_el0
+	sub	PTRN(sp), PTRN(sp), #16
+	mrs	c9, ddc_el0
+	str	c9, [PTRN(sp)]
+	adrp	PTR(9), kernel_root_cap
+	add	PTR(9), PTR(0), :lo12:kernel_root_cap
+	ldr	c16, [PTR(9)]
+	msr	ddc_el0, c9
```

Comment:
Wrong register, but good enough because PCC isn’t bounded

Reply:
Oh, that's just a silly bug from before.  I had used c16 originally.  I think it matters though as the base address of DDC would offset the accesses via SP_EL0.